extensions/amp-carousel/0.1/scrollable-carousel.js

```diff
@@ -89,6 +89,12 @@ export class AmpScrollableCarousel extends BaseCarousel {
       },
       ActionTrust.LOW
     );
+    /** If the element is in an email document, allow its `goToSlide` action. */
+    Services.actionServiceForDoc(this.element).addToWhitelist(
+      'AMP-CAROUSEL',
+      'goToSlide',
+      {isAllowedInEmail: true}
```

Comment:
Note that non-primitives that cross binary boundaries (e.g. extension calling a v0.js API) needs to be externed to avoid accidental property renaming due to minification. Another reason to prefer primitives here.